### PR TITLE
Use FilterChips in systems table

### PR DESCRIPTION
**Pull request by akofink**
Switch to using ConditionalFilter, as specified in inventory.md

Signed-off-by: Andrew Kofink <akofink@redhat.com>

### CODE REVIEW

File: src/SmartComponents/SystemsTable/SystemsTable.js [hunk 1]
```diff
@@ -141,33 +174,19 @@ class SystemsTable extends React.Component {
...
-                        <ComplianceRemediationButton />
-                    </reactCore.ToolbarItem>
-                    <reactCore.ToolbarItem style={{ marginLeft: 'var(--pf-global--spacer--md)' }}>
-                        <DownloadTableButton />
-                    </reactCore.ToolbarItem>
-                </reactCore.ToolbarGroup>
-            </InventoryCmp>
+                filter={filter}
+                filterConfig={filterConfig}
+                activeFiltersConfig={activeFiltersConfig}
```
Comment (dLobatog) [inline src/SmartComponents/SystemsTable/SystemsTable.js]: How does this work if DownloadTableButton, ComplianceRemediationButton and items are not in there?

Comment (akofink) [reply src/SmartComponents/SystemsTable/SystemsTable.js]: It doesn't right now - that's one reason why it's draft ;) But we can supposedly pass those buttons to PrimaryToolbar's actionsConfig prop.

File: src/SmartComponents/SystemsTable/SystemsTable.js [hunk 2]
```diff
@@ -46,11 +46,48 @@ query getSystems($filter: String!, $perPage: Int, $page: Int) {
...
+                            { label: '90 - 100%', value: '90-100' },
+                            { label: '70 - 89%', value: '70-89' },
+                            { label: '50 - 69%', value: '50-69' },
+                            { label: 'Less than 50%', value: '0-49' }
+                        ]
+                    }
+                }
+            ]
+        },
+        actionsConfig: {
```
Comment (akofink) [inline src/SmartComponents/SystemsTable/SystemsTable.js]: @dLobatog this prop here - I haven't figured it out yet, but it's possible to pass components such as our remediate button.

**Comment by akofink** [comment]
Closing as stale and broken :(

**Pull request closed without merging**